File: Module-CP/Assignments/assignment13/q9.c
#include<stdio.h>
#include<stdlib.h>
int main(){
    int s;
    printf("Enter Size Of array:");
    scanf("%d",&s);
    int*arr1=(int*)malloc(sizeof(int)*s);
    printf("\nEnter elements in array:");
    for(int i=0;i<s;i++){
        scanf("%d",&arr1[i]);
    }
    printf("\nArray:");
    for(int i=0;i<s;i++){
        printf("%d ",arr1[i]);
    }
    int start=0;int end=s-1;
        while(start<end){
            int temp=arr1[start];
            arr1[start]=arr1[end];
            arr1[end]=temp;
            start++;
            end--;
        }printf("\n Reversed Array:");
        for(int i=0;i<s;i++){
        printf("%d ",arr1[i]);
    }
    
}

File: Module-Core Java/Assignment 3/q9.java
public class q9{
    public static void main(String[] args) {
        int n=1221;
        int digit;
        int temp=n;
        int reversed=0;
        while(n>0){
            digit=n%10;
            reversed=(reversed*10)+digit;
            n/=10;
        }
        if(reversed==temp)
        System.out.println("palindrome");
        else
        System.out.println("Not palindrome");
    }
}